src/main/kotlin/day15/Repository/Todos.Repository.kt

```
@@ -0,0 +1,36 @@
+            return todo.filter { todo -> todo.completed }
+        } catch (e: Exception) {
+            throw CustomErrorHandler("Unexpected Error")
+        }
+    }
+
+}
+
+
+const val todoDataSource = "./src/main/kotlin/day15/RawData/todos.json"
+
+open class TodoRepository : TodoDataSourceImp(readJson(todoDataSource)) {}
```

detekt (1.23.7)

[warning] The class or object TodoRepository is empty.

(detekt.empty-blocks.EmptyClassBlock)